stockroom/utils.py

```diff
@@ -2,6 +2,39 @@
 import types
 from pathlib import Path
 
+from rich import box
+from rich.console import Console
+from rich.table import Table
+
+# init console object to be used throught.
+console = Console()
+
+
+def new_columns_table(splits_added: dict) -> Table:
```

Comment:
Use a better (distinguishable) name so that readers won't get confused this `column` with hangar column